src/test/java/revai/unit/MockClientListener.java

```diff
@@ -0,0 +1,32 @@
+package revai.unit;
+
+import okhttp3.Response;
+import revai.RevAiWebsocketListener;
+import revai.models.streaming.Hypothesis;
+
+public class MockClientListener implements RevAiWebsocketListener {
```

Comment:
minor, do you need this? cant you just mock it with mockito (which i think is already in your dependency list). eg you would just do `mock(RevAiWebsocketListener.class)` to get a mock instance.